### PR TITLE
base: add the sixteen states of Germany

### PR DESCRIPTION
**Pull request by booo**
Description of the issue/feature this PR addresses:

This pull request adds the sixteen states of Germany to the set of states provided by the base addon.

Current behavior before PR:

The sixteen states of Germany where missing in the list of states.

Desired behavior after PR is merged:

The sixten states of Germany should be present in the list of states by default.

Discussion: Maybe we should have an addon per country which populates the states instead?

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/res/res.country.state.csv [hunk 1]
```diff
@@ -677,3 +677,19 @@ state_co_30,co,"Guainía","GUA"
...
+state_de_bw,de,"Baden-Würtenberg","BW"
+state_de_by,de,"Bayern","BY"
+state_de_be,de,"Berlin","BE"
+state_de_bb,de,"Brandenburg","BB"
+state_de_hb,de,"Bremen","HB"
+state_de_hh,de,"Hamburg","HH"
+state_de_he,de,"Hessen","HE"
+state_de_mv,de,"Mecklenburg-Vorpommern","MV"
+state_de_ni,de,"Niedersachsen","NI"
+state_de_nw,de,"Nordrhein-Westfalen"
```
Comment (rim-odoo) [inline odoo/addons/base/res/res.country.state.csv]: I think you forgot a column with the "state" code in this line

Comment (booo) [inline odoo/addons/base/res/res.country.state.csv]: Thanks for the review. I updated the pull request with a fixed version.

**Comment by wtaferner** [comment]
@rim-odoo 
Wait, I thought it is only accepted in base if it is requested by law to give an information about the state or needed in the address to find?

If not, I will revamp my PR for Austria to get it in master as well -
 https://github.com/odoo/odoo/pull/13069

**Comment by mart-e** [comment]
Hello,

Could you please add it in the localization module instead of in base?

**Comment by booo** [comment]
I don't have the time right now. Therefore I close my pull request. Feel free to reuse my code.

**Comment by Yenthe666** [comment]
@booo I've reused your code and forward it to the `master` version under the right module. This will be available in the upcoming V12.

**Pull request closed without merging**